automation/src/test/scala/org/broadinstitute/dsde/test/api/WorkspacesAzureApiSpec.scala

```diff
@@ -43,14 +55,100 @@ class AzureWorkspacesSpec extends AnyFlatSpec with Matchers with CleanUp {
         Set.empty,
         Map("disableAutomaticAppCreation" -> "true")
       )
+      try {
+        val response = workspaceResponse(Rawls.workspaces.getWorkspaceDetails(projectName, workspaceName))
+        response.workspace.name should be(workspaceName)
+        response.workspace.cloudPlatform should be(Some(WorkspaceCloudPlatform.Azure))
+        response.workspace.workspaceType should be(Some(WorkspaceType.McWorkspace))
+      } finally {
+        Rawls.workspaces.delete(projectName, workspaceName)
```

Comment:
I just moved the workspace deletion into a finally clause-- select Hide Whitespace to see this.